src/modules/popUp.js

```diff
@@ -0,0 +1,268 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] KIndly remove this & solve the linter manually, if the error present.

Reply:
well resolved! ✔️